src/foam/dao/FileJournal.js

```diff
@@ -199,18 +200,32 @@ foam.CLASS({
       synchronized: true,
       javaCode: `
         try {
+          String c = "";
+          if ( multiLineOutput ) {
```

Comment:
I don't think we should toggle multLine on the Outputter every put operation.  That's unnecessary, I would just update the javaFactory for Outputter to read the local multilineOutput property and set it up appropriately once at creation time.